### DISCUSSION TITLE
I2c Clear status registers

### DISCUSSION THREAD

**Discussion by birdo3000** [electronicayciencia/EasyMCP2221#12] [Q&A] [answered]
Hello. I am trying to read status registers from my device and they don't clear faults automatically. The I2c chip LM5066I has a register for clearing status registers, but I cannot figure out how to invoke it as it is neither a read nor a write register.

![image](https://github.com/electronicayciencia/EasyMCP2221/assets/95247478/2d4ae274-cc37-4bd3-ac4f-ee7c648a2fe5)
![image](https://github.com/electronicayciencia/EasyMCP2221/assets/95247478/1d00ee47-edd8-4504-9755-bc490785b2ab)

I tried just using the read register function and the write register function putting in "CLEAR_FAULTS" but that doesn't seem to work.

Can anyone help me understand how I might make sure the clear faults register gets activated correctly? Thanks!

**Comment by electronicayciencia** [comment]
Hello. I've never worked with that chip. According with the datasheet, you just send the command 03h. No need to read or write any more data. Try the SMBus compatible class. If it doesn't work, try raw I2C commands to debug the issue.

**Reply by birdo3000** [reply]
Using this library, how do I send a command?

**Reply by birdo3000** [reply, marked as answer]
Oh I figured it out now. I need to use the smbus _write command and put the i2c address for address and the clear faults '0x03' for data.

this seems to work! thank you.
